packages/components/button/test/ToggleButton.test.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { Preview } from '@contentful/f36-icons';
+import { axe } from '@/scripts/test/axeHelper';
+
+import { ToggleButton } from '../src/ToggleButton';
+describe('ToggleButton', function () {
+  it('renders the component', () => {
+    const { container } = render(<ToggleButton>Toggle</ToggleButton>);
+
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  it('renders the component with an additional class name', () => {
+    const { container } = render(
+      <ToggleButton className="my-extra-class">Toggle</ToggleButton>,
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  it('renders the component active', () => {
+    const { container } = render(<ToggleButton isActive>Toggle</ToggleButton>);
+
+    expect(container.firstChild).toMatchSnapshot();
```

Comment:
a hot take, but don't we want to replace snapshots with queries verification? 

we could verify here that a particular aria attribute is present, for example

Reply:
uhmm thats true, I only kept like this as other components also used the snapshots tests too.